src/documentation/markdown.rs

```diff
@@ -55,6 +55,13 @@ impl<'a> PlainOverlay<'a> {
         }
     }
 
+    fn valid_url(url: &str) -> bool {
+        match url::Url::parse(url) {
+            Ok(_) => true,
+            Err(_) => false,
+        }
```

Comment:
```suggestion
        url::Url::parse(url).ok()
```

iirc this should work as well.